plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/MinioClient.java

```diff
@@ -45,7 +45,7 @@
 import static java.util.Objects.requireNonNull;
 import static java.util.regex.Matcher.quoteReplacement;
 
-class MinioClient
+public class MinioClient
```

Comment:
NIT: When we use MinioClient now directly in container. I'm wondering if we could maybe move client creation to MinIo container and fetch it from them anywhere it's needed ?